### PR TITLE
PRIME-1666 Paper Enrolment Updates

### PR DESCRIPTION
**Pull request by mtpultz**

### CODE REVIEW

File: prime-angular-frontend/src/app/lib/utils/array-utils.class.spec.ts [hunk 1]
```diff
@@ -64,4 +64,20 @@ describe('ArrayUtils', () => {
       expect(result).toEqual(['Yosemite Sam', null, 'Road Runner']);
     });
   });
+
+  describe('intersection', () => {
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/lib/utils/array-utils.class.spec.ts:68]: Didn't mean to commit but too late

File: prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.ts [hunk 2]
```diff
@@ -174,10 +174,12 @@ export class CollegeCertificationFormComponent implements OnInit {
           }
         });
 
+      const initialNursingCategory: number | null = +this.nurseCategory.value ?? null;
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.ts:177]: Fix for nursing category not loading on update

**Review by hogmann — APPROVED** [review]
LGTM 🐄

**Review by hogmann — DISMISSED** [review]
LGTM 🐴

**Review by hogmann — DISMISSED** [review]
LGTM 🐑

**Review by hogmann — APPROVED** [review]
LGTM 🦌

**Review by james-hollinger — APPROVED** [review]
L LikeIWorkedOnThisPRAndImApprovingMyOwnWork TM

**Pull request merged**